### PR TITLE
CSS-564 Candid branding.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -130,6 +131,12 @@ type Config struct {
 
 	// MFARPOrigin holds the relying party origin for MFA.
 	MFARPOrigin string `yaml:"mfa-rp-origin"`
+
+	// BrandName holds the name of the entity running candid.
+	BrandName string `yaml:"brand-name"`
+	// BrandLogoLocation holds the location of the logo of the entity
+	// runnin candid.
```
Comment (TeodorPt) [inline config/config.go]: nit: running

File: templates/authentication-required [hunk 2]
```diff
@@ -17,8 +17,13 @@
 <body cz-shortcut-listen="true">
   <div class="p-strip">
     <div class="logo">
-      <img class="logo__image" src="static/images/logo-canonical-aubergine.svg" alt="Canonical" width="480" height="65" />
+      <img class="logo__image" src="/static/images/logo-canonical-aubergine.svg" alt="Canonical" width="480" height="65" />
```
Comment (TeodorPt) [inline templates/authentication-required:20]: intentional?

**Review by TeodorPt — APPROVED** [review]

File: templates/authentication-required (hunk 2 above)
Comment (alesstimec) [reply templates/authentication-required:20]: yup

File: config/config.go (hunk 1 above)
Comment (alesstimec) [reply config/config.go]: fixed, thanks

**Pull request merged**